src/yarn-frameworklauncher/deploy/yarn-frameworklauncher.yaml.template

```diff
@@ -62,6 +62,13 @@ spec:
           value: {{ clusterinfo[ 'frameworklauncher' ][ 'frameworklauncher_vip' ] }}
         - name: FRAMEWORKLAUNCHER_PORT
           value: "{{ clusterinfo[ 'frameworklauncher' ][ 'frameworklauncher_port' ] }}"
+        - name: LAUNCHER_OPTS
```

Comment:
Comment here 4Gi + -Xmx3072m -Djute.maxbuffer=49107800 can hold how many jobs and tasks

Reply:
Good point! I will collect some info.